diamond_agent/resources/diamond

```diff
@@ -26,7 +26,7 @@ case "$1" in
     start)
     export CELERY_WORK_DIR="{{ WORK_DIR }}"
     export CLOUDIFY_CLUSTER_SETTINGS_PATH="{{ CLUSTER_SETTINGS_PATH }}"
-    {{ CMD }}
+    sudo su {{ AGENT_USER }} -c "{{ CMD }}"
```

Comment:
can we just `sudo -u {{ AGENT_USER }} {{ CMD }}`?

Reply:
That would be better.